### DISCUSSION TITLE
reload4j support

### DISCUSSION THREAD

**Discussion by brunovianarezende** [getsentry/sentry-java#2214] [Q&A] [answered]
Hi, I have a project which uses reload4j, i.e. a compatible log4j 1.x library. Newest sentry versions don't have support for log4j 1.x and I need to use the legacy SDK, which prevents me from doing things like filtering events (https://docs.sentry.io/platforms/java/configuration/filtering/#using-platformidentifier-namebefore-send-). Is there any plans to re-add log4j 1.x support now that reload4j keeps the library alive?

**Comment by adinauer** [comment, marked as answer]
Hello @brunovianarezende, currently there are no plans on adding back support for log4j 1.x. If we see enough people asking for it, we could prioritize adding it back in.
